pkg/util/trace/schema.go

```diff
@@ -0,0 +1,112 @@
+package trace
+
+import (
+	"fmt"
+	"github.com/matrixorigin/matrixone/pkg/logutil/logutil2"
+	"time"
+
+	ie "github.com/matrixorigin/matrixone/pkg/util/internalExecutor"
+)
+
+const (
+	systemDBConst    = "system"
+	statsDatabase    = systemDBConst
+	spanInfoTbl      = "span_info"
+	logInfoTbl       = "log_info"
+	statementInfoTbl = "statement_info"
+	errorInfoTbl     = "error_info"
+)
+
+const (
+	sqlCreateDBConst = `create database if not exists ` + statsDatabase
+
+	sqlCreateSpanInfoTable = `CREATE TABLE IF NOT EXISTS span_info(
+ span_id BIGINT UNSIGNED,
```

Comment:
span_id and log_info id, they are surrogate keys?    I am not a big fan of this thing.    How would people use this?

Is session id, statement id node_id etc, plus timestamp enough?    Most likely we don't need any primary key for log, and not span_id as foreign key as well.    Most likely we just want to scan this thing.

Reply:
log_info id is useless, will be removed.

Reply:
in table log_info, span_id is NOT necessary, but can be related to a specific span.
in table span_info, span_id and parent_span_id are `necessary` to build like a call-stack tree.